hamster_lib/objects.py

```diff
@@ -468,7 +479,18 @@ def comma_split(string):
         activity_name, back = at_split(rest)
 
         if back:
-            category_name, description = comma_split(back)
+            # Check for an existing hashtag.
```

Comment:
This is an interesting approach. The next iteration of ``hamster-lib`` will however refactor the entire 'raw fact' parsing machinery in total, which is why no effort has been made to take care of tags.